llvm/lib/Transforms/Vectorize/VPlanPatternMatch.h

```diff
@@ -254,7 +248,6 @@ struct Recipe_match {
     // Check for recipes that do not have opcodes.
     if constexpr (std::is_same<RecipeTy, VPScalarIVStepsRecipe>::value ||
                   std::is_same<RecipeTy, VPCanonicalIVPHIRecipe>::value ||
-                  std::is_same<RecipeTy, VPWidenSelectRecipe>::value ||
```

Comment:
This was removed because `VPWidenSelectRecipe` was added to `AllRecipe_match` (to remove `TernaryRecipe_match`), and we don't want unary or binary matchers to unconditionally match on `VPWidenSelectRecipe`. 

`VPWidenSelectRecipe` defines getOpcode() to be Instruction::Select so just use that instead.